### PR TITLE
Bump ormolu to 0.1.2.0

### PR DESCRIPTION
**Pull request by AlistairB**
hls tests are passing. The ghcide tests don't pass for me on master.

### CODE REVIEW

File: haskell-language-server.cabal [hunk 1]
```diff
@@ -80,7 +80,7 @@ library
     , hie-bios >= 0.4
     , hslogger
     , lens
-    , ormolu ^>= 0.0.5.0
```
Comment (AlistairB) [inline haskell-language-server.cabal:83]: Wasn't sure what to set this to exactly. Not sure if this is correct?

File: stack-8.8.3.yaml [hunk 2]
```diff
@@ -28,7 +28,7 @@ extra-deps:
 - lsp-test-0.11.0.2
 - monad-dijkstra-0.1.1.2
 - opentelemetry-0.4.2
-- ormolu-0.0.5.0
+- ormolu-0.1.2.0
```
Comment (AlistairB) [inline stack-8.8.3.yaml:31]: `lts-16.3` actually has `ormolu-0.1.2.0`. However, I'm guessing you want to pin this here so you can bump the stackage version without bumping ormolu? Or should I remove this line?

File: haskell-language-server.cabal [hunk 3]
```diff
@@ -80,7 +80,7 @@ library
...
+    , ormolu ^>= 0.1.2
```
Comment (AlistairB) [inline haskell-language-server.cabal:83]: Wasn't sure what to set this to exactly. Not sure if this is correct?

Comment (fendor) [reply haskell-language-server.cabal:83]: The caret operator is special syntax to add a PVP conform version constraint.
IIRC, in PVP in the version a.b.c.d, `a.b` are major versions while `c.d` are minors, so should be compatible.
Therefore, this syntax is identical to this constraint (iirc):
```cabal
  , ormolu >= 0.1.2 && < 0.2
``` 
So, yeah that is correct

File: stack-8.8.3.yaml (hunk 2 above)
Comment (fendor) [reply stack-8.8.3.yaml:31]: Yeah that is fine! Thank you!

**Pull request merged**